e2e/testsuite/testsuite.go

```diff
@@ -15,6 +15,8 @@ import (
 	testifysuite "github.com/stretchr/testify/suite"
 	"go.uber.org/zap"
 
+	sdkmath "cosmossdk.io/math"
```

Comment:
The changes in this filed are required because of the bump of interchaintest.

Reply:
looks like there's a compiler error in "build e2e" workflow from this atm 👀

Reply:
Ay, I will check!